contracts/DolaFlashMinter.sol

```diff
@@ -73,26 +77,20 @@ contract DolaFlashMinter is Ownable, IERC3156FlashLender {
     }
 
     function setTreasury(address _newTreasury) external onlyOwner {
+        require(_newTreasury != address(0), "FLASH_MINTER:INVALID_TREASURY");
         emit TreasuryUpdated(treasury, _newTreasury);
         treasury = _newTreasury;
     }
 
-    function maxFlashLoan(address _token)
-        external
-        view
-        override
-        returns (uint256)
-    {
+    function maxFlashLoan(address _token) external view override returns (uint256) {
         return _token == address(dola) ? type(uint112).max - flashMinted : 0;
     }
 
-    function flashFee(address _token, uint256 _value)
-        public
-        view
-        override
-        returns (uint256)
-    {
-        require(_token == address(dola), "!dola");
+    function flashFee(address _token, uint256 _value) public view override returns (uint256) {
+        require(_token == address(dola), "FLASH_MINTER:NOT_DOLA");
         return (_value * flashLoanRate) / 1e18;
     }
+
+    // solhint-disable-next-line no-empty-blocks
```

Comment:
Maybe we should ignore this lint warning in the `.solhint.json` rather than paste this line in every file

Reply:
I prefer to keep warning exclusion scope tight, can change if really necessary.